tests/api/test_devicespec.py

```diff
@@ -46,6 +46,15 @@
     },
 }
 
+device_certificate = {
+    "target": "X8_01",
+    "finished_at": "2022-02-07T15:29:48.080220+00:00",
+    "identity_unitary": {},
+    "shots_per_benchmark": 999999,
+    "anti_identity_unitary": {},
+    "laser_wavelength_meters": 42,
+}
```

Comment:
Will all device certificates have these fields? Or is this just an example of possible fields?